bugsnag-android-core/src/main/java/com/bugsnag/android/internal/ConcurrentContainers.kt

```diff
@@ -0,0 +1,57 @@
+package com.bugsnag.android.internal
+
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.ConcurrentMap
+import java.util.concurrent.CopyOnWriteArrayList
+
+/**
+ * Utility to convert containers to their concurrent counterparts.
+ */
+class ConcurrentContainers private constructor() {
+    companion object {
```

Comment:
```suggestion
object ConcurrentContainers {
```

Alternatively - these might make nice extension functions? @fractalwrench ?

Reply:
Extension functions sounds good - all the symbols should also remain `internal`